cmd/pyroscope/help-all.txt.tmpl

```diff
@@ -467,8 +467,14 @@ Usage of ./pyroscope:
     	Minimum time to wait for ring stability at startup, if set to positive value. Set to 0 to disable.
   -pyroscopedb.data-path string
     	Directory used for local storage. (default "./data")
+  -pyroscopedb.enforcement-interval duration
+    	How often to enforce disk retention (default 5m0s)
```

Comment:
```
-pyroscopedb.enforcement-interval
```
Should the name be more specific, like `retention-policy-enforcement-interval`? Maybe we also could keep this prefix for all the related settings for better discoverability